api/model/review/ReviewModel.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
+
+  rating: {
+    type: Number,
+    required: true,
+    min: 1,
+    max: 5,
+  },
+  comment: {
+    type: String,
+    required: true,
+    trim: true,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
```

Comment:
May be you are missing the product yhat you are reviewing, pelase check?

Reply:
Added Product.